scripts/dotnet-cli-build/TestTargets.cs

```diff
@@ -178,7 +165,7 @@ public static BuildTargetResult BuildTestAssetPackages(BuildTargetContext c)
                 {
                     "netstandard1.5",
                     "netstandard1.3",
-                    "netstandardapp1.5"
+                    "netcoreapp1.0"
```

Comment:
We build the projects on all of these frameworks so we can ignore net451 failures on non-windows.

The package is only built once, with or without net451 binaries depending on platform.

The test assets that use these packages need to be restored against them just after this step, so it would be complex (and ugly) to orchestrate that in individual tests.